src/constants/prop-types/img-with-placeholder.js

```diff
@@ -1,9 +1,9 @@
 import PropTypes from 'prop-types'
 
 const imagePropType = PropTypes.shape({
-  url: PropTypes.string.isRequired,
-  width: PropTypes.number.isRequired,
-  height: PropTypes.number.isRequired,
+  url: PropTypes.string,
+  width: PropTypes.number,
+  height: PropTypes.number,
```

Comment:
If this `propType` is for `img-with-placeholder`, it's better to keep `isRequired` for these properties since we need `width` and `height` to calculate the aspect ratio.

Reply:
`img-with-placeholder` will `console.warn` if `width` and `height` are not provided.

The reason why I remove `isRequired` here is because `prop-types` will `console.log` the code stack on the server standard output.